clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -143,6 +143,33 @@ public static class Params extends Command.Params {
     // Debug run.
     private boolean debug = false;
 
+    public Params() {
```

Comment:
Quick question about the design of these param classes:
I assume these attributes should be immutable after the initialization, right?
Shall we use a `builder` pattern here instead of exposing `setter` methods?

Reply:
Introduced builder pattern in newest iteration.